eigrpd/eigrpd.h

```diff
@@ -52,4 +52,33 @@ extern struct eigrp *eigrp_get(const char *);
 extern struct eigrp *eigrp_lookup(void);
 extern void eigrp_router_id_update(struct eigrp *);
 
+/* eigrp_cli.c */
+extern void eigrp_cli_show_header(struct vty *, struct lyd_node *, bool);
```

Comment:
Please put variable names in the declarations here.  Otherwise this all looks good.

Reply:
Fixed!